docs/validators/delegation-manager.mdx

```diff
@@ -136,43 +136,84 @@ Setting the identity of the staking pool in the metadata is the **first step** i
 
 [comment]: # (mx-context-auto)
 
+### Display information
+
 :::caution
-As of March 2023, keybase.pub is no longer available. As a consequence, the identity verification mechanism has been switched to GitHub, including the profile information fetched from keybase.io.
+As of January 2024, the only accepted way to customize the information of a delegation contract is via MultiversX Assets.
+Provisioning of information from keybase or your Github repository is no longer accepted and you should migrate to the new approach is you still use those.
```

Comment:
"
Provisioning of information from keybase or your Github repository is no longer accepted.
The only accepted changes are on `mx-assets` identities.
You can safely delete the keybase profile and also the `multiversx` repository inside your organization's github.
"

Reply:
done